config/releases.exs

```diff
@@ -14,18 +14,32 @@ live_view_salt =
     You can generate one by calling: mix phx.gen.secret
     """
 
+app_name = System.get_env("FLY_APP_NAME", "elixirschool")
+
 host =
-  case System.get_env("HEROKU_APP_NAME") do
-    nil -> "elixirschool.com"
-    sub -> "#{sub}.herokuapp.com"
+  case app_name do
+    "elixirschool" -> "elixirschool.com"
```

Comment:
I tried to be a little clever here, so someone could possibly just deploy this app on their own and the name would auto fill correctly. The down side is if you create the `elixirschool` app, and don't access it with `elixirschool.com`, you'll get a socket origin warning and live view will just keep refreshing the page.